app/assets/stylesheets/nucore.scss

```diff
@@ -377,3 +377,16 @@ textarea.wide {
 .checkboxControl__checkbox {
   margin-right: 2.5em;
 }
+
+.productListLine {
+  display: flex;
+}
+
+.productListLine__productName {
```

Comment:
I'm not quite sure what these do, but would `productListLine__first` and `last` be easier to work with in the future?